app/controllers/UserController.scala

```diff
@@ -232,6 +269,7 @@ class UserController @Inject()(userService: UserService,
         count <- userDAO.countIdentitiesForMultiUser(user._multiUser)
         issuingMultiUser <- multiUserDAO.findOne(issuingUser._multiUser)
         _ <- Fox.fromBool(count <= 1 || issuingMultiUser.isSuperUser) ?~> "user.email.onlySuperUserCanChange"
+        // TODOM: @fm3 should we keep this check as now we can have guest users?
```

Comment:
If my understanding is correct, admins can no longer change their user’s email, but instead users can only change their own, right?.

If so, being in multiple organizations should no longer be a problem for this. This was supposed to prevent orga admins making changes that affect other orgas.

 So the check (and the whole function) can go.